Validation/RecoTau/python/dataTypes/ValidateTausOnQCD_cff.py

```diff
@@ -25,8 +25,23 @@
 zttModifier = ApplyFunctionToSequence(zttLabeler)
 proc.TauValNumeratorAndDenominatorQCD.visit(zttModifier)
 
+#Set discriminators
+discs_to_retain = ['ByDecayModeFinding', 'CombinedIsolationDBSumPtCorr3Hits', 'IsolationMVArun2v1DBnewDMwLT', 'IsolationMVArun2v1PWnewDMwLT']
```

Comment:
Why we retain MVAIso discriminants for newDMs, but not for oldDMs which are wider used? And why we do not test ByDecayModeFindingNewDMs which looks inconsistent with tested MVAIso?